requirements/base.txt

```diff
@@ -10,6 +10,7 @@ scipy==1.7.3
 overrides==3.1.0
 jsonschema==3.2.0
 tenacity==7.0.0
+twine==3.7.1
```

Comment:
Since we're using the dev docker image, we can add this requirement to `dev.txt` instead of `base.txt`